py3dtilers/Texture/atlas.py

```diff
@@ -65,7 +65,7 @@ def computeAtlasTree(self, textures_sorted):
         # where this does not work, since this creates a square.
         sizeOfAtlas = self.multipleOf2(np.sqrt(surfaceAtlas))
 
-        rect = Rectangle(0, 0, (sizeOfAtlas // 2) + 1, (sizeOfAtlas // 2) + 1)
+        rect = Rectangle(0, 0, sizeOfAtlas, sizeOfAtlas)
```

Comment:
Are you sure that when we extend the rectangle (line 84-85), it is kept as POT ? (did not make check it myself, just wanted to raise those lines)

Reply:
I'll check that

Reply:
Should be fine now